### PR TITLE
Fixes issue #769

### PR DESCRIPTION
**Pull request by Remorax**
Fixes issue #769 
Design changes have been made as given here: http://yacy.searchlab.eu/CrawlStartExpert.html
Changes:
1. The entire form was put in a fieldset with legends, and this was further divided into fieldsets as given on the above mentioned link.
2. The radio buttons of the start point section were left aligned.
3. The labels were made bold as given in the above link
4. The legends have green background, again as per the link
5. Other than the above, the page has been made uncluttered. In some cases, the checkboxes and radio buttons have been put on separate lines, and the text beside them has been given margin for the same purpose.

Screenshots for the change: 
Screenshots after the above changes were made:

screen shot 2017-10-31 at 2 43 18 am
screen shot 2017-10-31 at 2 43 26 am
screen shot 2017-10-31 at 2 43 34 am

@harshit98 @mariobehling Please review :)

### CODE REVIEW

**Comment by harshit98** [comment]
@Remorax Can you share any demo link for it? A lot of changes has been done in it. Good Work! 🎉 Please keep the background color white only and provide a demo link for quick review.

For generating a preview link you can use:
- `ng build`
- `npm run deploy`

Make sure you are deploying your `crawlpage-design-changes`

**Comment by Remorax** [comment]
@harshit98 This is the demo link: 
https://remorax.github.io/susper.com/crawlstartexpert

**Comment by harshit98** [comment]
Looks great @Remorax. Please change the background color to white instead of using grey.

**Comment by Remorax** [comment]
@harshit98 I've made the requested changes. You can check it on the same link. Please accept it :)

File: deploy.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-SOURCE_BRANCH="master"
+SOURCE_BRANCH="crawlpage-design-changes"
 TARGET_BRANCH="gh-pages"
```
Comment (harshit98) [inline deploy.sh]: Revert back the changes here please.

**Comment by harshit98** [comment]
@Remorax I'm observing there are 4 commits in this PR. According to best practices, please squash them. The PR looks good to me. You have done a great job! Please go through the suggested changes for approval.

**Comment by Remorax** [comment]
Can I close this and make another PR? I'm unable to figure out what to do. A lot of commits have gotten mixed together, and it's too complicated for me. Sorry :)

**Comment by harshit98** [comment]
No need to close it. I'm approving the PR. You can ask me anytime if you face any difficulty on gitter channel.

**Comment by Remorax** [comment]
@mariobehling, please review :)

**Comment by mariobehling** [comment]
Please also try to get other contributors to review your changes to help to speed up the review process. Keep it up! ;-)

**Comment by bhaveshAn** [comment]
Is this issue still active ? As I would like to work on this...

**Comment by Remorax** [comment]
No, sorry I've already worked on this, and a few changes are required which I'll be making shortly, since I've been busy the past few days. The whole page has been restructured @bhaveshAn

**Comment by Remorax** [comment]
I'm closing this PR as a lot of changes have been made, and I'm unable to squash them. I'll be creating another PR shortly

**Pull request closed without merging**